diff --git a/script/canvas_init b/script/canvas_init
index 79d2b0ea..5fb7d2b1 100755
--- a/script/canvas_init
+++ b/script/canvas_init
@@ -13,7 +13,7 @@ set -e
 
 # drop privs if necessary
 if [ "$(id -u)" == "0" ]; then
-  exec su $(stat -c %U $(dirname $(readlink -f $0))/../config/environment.rb) -c "/bin/bash $0 $@"
+  exec su $(stat -c %U $(dirname $(readlink -f $0))/../config/environment.rb) -c "/bin/bash $0 $@" -s /bin/bash
   exit -1;
 fi
 
@@ -26,6 +26,7 @@ if [ -e "config/GEM_HOME" ]; then
 fi
 #export GEM_HOME=/path/to/gem/home
 export RAILS_ENV=production
+export BUNDLE_PATH="vendor/bundle"
 
 # run delayed jobs
-exec script/delayed_job $@
+bundle exec script/delayed_job $@
